javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -11,55 +11,49 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 When working with packages that are installed with npm, you don't need to track the contents of `node_modules` with git, nor push those files to GitHub. This is because the `package.json` file contains all the information, so that anyone can clone your project and install them on their machine with `npm install`.
 
 You can make a `.gitignore` file in the root of the project, and by writing file or directory names in it, you can tell git what things you don't want to track. It's customary to add `node_modules` to `.gitignore`, since it can get really big.
+
 </div>
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
 1. Start the project the same way you began the webpack tutorial project.
-    1. Run `npm init` in your project directory to generate a `package.json` file.
```

Comment:
It may be worth having the student go back to the previous lesson to set up the environment in a new folder. This will negate the need to repeat the instructions in this assignment. Therefore, the actual assignment can start from the new content from line 36?